### PR TITLE
Update helm chart

### LINKED ISSUE #199: Helm chart is outdated
**Describe the bug**
The Helm chart ([manifests/charts/aeraki](https://github.com/aeraki-mesh/aeraki/tree/1879f821ecea8d6e61871abbfc307242482f6b77/manifests/charts/aeraki)) is outdated. It needs to be updated so we can use it to install aeraki.

### PR DESCRIPTION
**Pull request by nevermosby**
Signed-off-by: Wen-Quan Li [legendarilylwq@gmail.com](mailto:legendarilylwq@gmail.com)

### What this PR does / Why we need it:

fix #199: update helm chart

### Pre-Submission Checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added related test cases?
* [x] Have you modified the related document?
* [x] Is this PR backward compatible?

### CODE REVIEW

File: manifests/charts/aeraki/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Copyright Aeraki Authors
```
Comment (zhaohuabing) [inline manifests/charts/aeraki/templates/configmap.yaml:1]: We don't need this configMap in the control plane installation. It's just for initial sidecar configuration.

**Review by zhaohuabing — CHANGES_REQUESTED** [review]

File: manifests/charts/aeraki/templates/configmap.yaml (hunk 1 above)
Comment (nevermosby) [reply manifests/charts/aeraki/templates/configmap.yaml:1]: Gotcha

**Comment by nevermosby** [comment]
@zhaohuabing Any idea how to fix the yaml lint error below?  Or just ignore?

```
======================
= Linting YAML files =
======================
./manifests/charts/aeraki/templates/tcmapiservice.yaml
  16:3      error    syntax error: expected the node content, but found '-' (syntax)
```

**Comment by zhaohuabing** [comment]
> @zhaohuabing Any idea how to fix the yaml lint error below? Or just ignore?
> 
> ```
> ======================
> = Linting YAML files =
> ======================
> ./manifests/charts/aeraki/templates/tcmapiservice.yaml
>   16:3      error    syntax error: expected the node content, but found '-' (syntax)
> ```

You can ignore the format checking by adding a line here: https://github.com/aeraki-mesh/aeraki/blob/2145283042e4429e1943db4eac6a1ff916d935d4/.ci/yamllint.yml#L4

But please do fix the "no new line character at the end of file" error

**Review by zhaohuabing — APPROVED** [review]
LGTM

**Pull request merged**